mono/mini/debugger-agent.c

```diff
@@ -4470,6 +4637,29 @@ set_breakpoint (MonoMethod *method, long il_offset, EventRequest *req, MonoError
 	method_seq_points = g_ptr_array_new ();
 
 	mono_loader_lock ();
+#ifdef IL2CPP_MONO_DEBUGGER
+	void *seqPointIter = NULL;
+	Il2CppSequencePointC *seqPoint;
+	while(seqPoint = il2cpp_get_sequence_points(&seqPointIter))
```

Comment:
Does this iterate all the sequence points?

Reply:
Yes. I suspect it is pretty inefficient. Maybe we could do this better by building a map from the method pointer to a list of sequence points. The code in `bp_matches_method` is not trivial though, so the key cannot be a pointer alone.

Reply:
I've added an item to our Trello backlog to track this issue. I'd like to land this PR without fixing this.

Reply:
To be honest, any lag felt debugging is probably trumped by how visual studio with resharper usually behavies ;p